development-process.md

```diff
@@ -0,0 +1,144 @@
+# ODL Development Process
+An approximate workflow guide to hacking on software at the MIT Office of Digital Learning.
+
+# Purpose
+Every team of developers has its own particular approach to writing, reviewing, and deploying code.
+This guide aims to capture MIT ODL's way of doing things. It will be (hopefully) useful as a
+step-by-step guide for new developers.
+
+# Tools
+
+- git
+- Zenhub Chrome extension (https://www.zenhub.io/)
+  - This extension adds the **Boards** and **Burndown** tabs to Github repos. We use it in the way
+  that you might use Jira or Trello. The **Boards** tab is particularly important as it adds a
+  'swim lane'-type view for open issues.
+
+# Coding Workflow
+
+
+### Step 1) Hacking
+
+#### &#10146; Clone the given repository from Github
+
+Example: `git clone https://github.com/mitodl/micromasters.git`
```

Comment:
Can you change this to the `git://` address? The `https` one is readonly I think